README.md

```diff
@@ -8,19 +8,29 @@ Copy all of the contents within the zip file into your Killing Floor directory.
 
 ## Usage
 
-Select the Killing Floor 2.5 Gametype and start a game.
+Select the Killing Floor 2.5 Gametype and start a game. Or for a dedicated server:
```

Comment:
Need to expand on the dedicated server portion afterwards. The additional formatting is nice.

Reply:
I reformatted the message a bit, but believe me server admins know what this means and where to use that line :d